### DISCUSSION TITLE
How to use a configuration stored in some custom directory as a default configuration?

### DISCUSSION THREAD

**Discussion by su66uu** [jrnl-org/jrnl#1596] [Q&A]
I am trying to use same configuration over two different user accounts on my Macbook. 
For this, I have moved the configuration file to `/Users/Shared` folder. I can use alternative config by using `jrnl --config-file [configuration file path]`.

But I want to use this new configuration file as a default so that when type `jnrl` on my terminal, it will directly consider my new configuration file path.

**Comment by micahellison** [comment]
Hi @svenkat683, I think the best approach is to use an alias that adds your `--config-file` arguments to a jrnl call. For instance, in bash, this would allow you to use that config file with a new "myjrnl" command:
```
alias myjrnl='jrnl --config-file /Users/Shared/jrnl.yaml'
```
To set that alias for all users, [see this Stackoverflow thread](https://stackoverflow.com/questions/30702803/set-aliases-globally-for-all-users). Or just run that alias command for each user.

There is another approach that would involve changing your XDG configuration folder by modifying the XDG_CONFIG_HOME environment variable as specified in the [XDG spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), but it would affect other applications that follow this spec, so I think the alias is a more solid approach.

**Reply by su66uu** [reply]
Hi @micahellison, I am following this alias approach already. Thank you. 
But is there any way or plan for setting up this using `jnrl`?

**Reply by micahellison** [reply]
Hi @svenkat683, no, the only way I can think of to add additional support for this in jrnl directly would be adding an extra environment variable for identifying the config, which I don't think is a good design decision in the long term.

However, I was talking @wren about this and he suggested using symlinks - keep a `jrnl` folder in your shared directory with `jrnl.yaml` in it, then in each user's default config folder, create a `jrnl` symlink to the `jrnl` folder in the shared directory.
